### DISCUSSION TITLE
Would like to remove image in previous knot - how?

### DISCUSSION THREAD

**Discussion by Beverly-Peders** [inkle/ink#687] [Writing in ink] [answered]
I'm designing a really stupid simple prototype game. Nothing really fancy. I have a grid map and a dot that moves on the map depending on the player choosing an available direction: up, down, left, right. 

I got the images to work, but once they choose the next direction, it keeps the old image and soon they have this whole string of images and it looks really bad. 

Is there a way I can tell the code to remove the image once they leave that knot? Like how when you have choices in [brackets] that text doesn't follow into the next knot. 

Thanks for your help.

**Comment by Selsynn** [comment]
I believe you'll have to do it on your own.
You can hook it at the same place the image is added, just remove every other previous images already present. (adding a class to the image to add/clear that way would be safer)

To help you more, we'll need to know what you are exporting to. I suppose it's web, because I don't believe image works out of the box for unity, but I could be wrong.

**Comment by Beverly-Peders** [comment, marked as answer]
Hello everyone, 

While working on my most recent project, I figured this out! It helps with clearing out the information from the previous knots, not just images!!! It's very useful! 

To give credit where credit is due, I learned it from this blog: https://videlais.com/2018/08/31/ink-for-the-web-part-3-meta-instructions/

Place # CLEAR (ALL CAPS is important!) on the line where you'd like to have the previous information be cleared. This includes images! For example: 

```
=== knot ===

# CLEAR 

Words
* choice 1
* choice 2

=== choice 1 ===

# CLEAR
```

I liked to use it at the top of knots so that the previous information doesn't clutter. Hope this helps!
